### PR TITLE
docs: Installation guide outline changes for devworkspace only release

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change

doc: Installation guide outline changes for devworkspace only release

1. Identify documentation subject to removal by commenting out sections in the outline. Validation by @l0rd @nickboldt. Reach consensus.
2. Cleanup content. Delete obsolete files. Handle redirections.

## What issues does this pull request fix or reference

https://github.com/eclipse/che/issues/21271

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/installation-guide/nav.adoc [hunk 1]
```diff
@@ -10,28 +10,21 @@
...
 *** xref:installing-che-on-openshift-4-using-cli.adoc[]
-*** xref:installing-che-on-openshift-3-using-the-operator.adoc[]
-*** xref:installing-che-on-kubespray.adoc[]
-*** xref:installing-che-on-aws.adoc[]
-*** xref:installing-che-on-google-cloud-platform.adoc[]
-*** xref:installing-che-on-microsoft-azure.adoc[]
-
+// *** xref:installing-che-on-openshift-3-using-the-operator.adoc[] // REMOVE
+// *** xref:installing-che-on-kubespray.adoc[] // UNMAINTAINED
+// *** xref:installing-che-on-aws.adoc[] // UNMAINTAINED
```
Comment (nickboldt) [inline modules/installation-guide/nav.adoc]: ROSA / AWS is important use case. We need this doc for CRW (but not sure if Che too).

File: modules/installation-guide/nav.adoc [hunk 2]
```diff
@@ -10,28 +10,21 @@
...
+// *** xref:installing-che-on-google-cloud-platform.adoc[] // UNMAINTAINED
+// *** xref:installing-che-on-microsoft-azure.adoc[] // UNMAINTAINED
 ** xref:running-che-locally.adoc[]
-
 *** xref:installing-che-on-codeready-containers.adoc[]
-*** xref:installing-che-on-docker-desktop.adoc[]
+// *** xref:installing-che-on-docker-desktop.adoc[] // UNMAINTAINED
 *** xref:installing-che-on-minikube.adoc[]
```
Comment (nickboldt) [inline modules/installation-guide/nav.adoc:19]: upstream only

File: modules/installation-guide/nav.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .Installation Guide
 
-* xref:supported-platforms.adoc[]
+* xref:supported-platforms.adoc[] // NEED REWRITE
```
Comment (nickboldt) [inline modules/installation-guide/nav.adoc:3]: upstream: k8s
downstream: openshift 4.10, osd 4.10

File: modules/installation-guide/nav.adoc [hunk 4]
```diff
@@ -10,28 +10,21 @@
 ** xref:using-chectl-to-configure-the-checluster-custom-resource-during-installation.adoc[]
 ** xref:using-the-cli-to-configure-the-checluster-custom-resource.adoc[]
 ** xref:checluster-custom-resource-fields-reference.adoc[]
```
Comment (nickboldt) [inline modules/installation-guide/nav.adoc:12]: this likely needs a rewrite.

File: modules/installation-guide/nav.adoc (hunk 1 above)
Comment (themr0c) [reply modules/installation-guide/nav.adoc]: Here, the kops tool is used to install Kubernetes. -> OT for downstream

**Review by l0rd — APPROVED** [review]

File: modules/installation-guide/nav.adoc [hunk 5]
```diff
@@ -41,30 +34,30 @@
...
 ** xref:mounting-a-secret-as-a-file-or-an-environment-variable-into-a-container.adoc[]
-** xref:enabling-dev-workspace-operator.adoc[]
+// ** xref:enabling-dev-workspace-operator.adoc[] // REMOVE BECOME DEFAULT
 
 * xref:upgrading-che.adoc[]
 
 ** xref:upgrading-che-using-operatorhub.adoc[]
 ** xref:upgrading-che-using-the-cli-management-tool.adoc[]
 ** xref:upgrading-che-using-the-cli-management-tool-in-restricted-environment.adoc[]
 ** xref:upgrading-che-namespace-strategies-other-than-per-user.adoc[]
```
Comment (l0rd) [inline modules/installation-guide/nav.adoc]:
```suggestion
// ** xref:upgrading-che-namespace-strategies-other-than-per-user.adoc[] // REMOVE
```

**Review by nickboldt — APPROVED** [review]

**Comment by dkwon17** [comment]
Hi @l0rd , was [this page](https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.15/html/end-user_guide/navigating-codeready-workspaces_crw#importing-certificates-to-browsers_crw) removed for the Dev Spaces docs because of the keycloak decommissioning? Or were there other reasons?

I am asking to figure out if [this error](https://issues.redhat.com/browse/CRW-3024) is still relevant today or not

**Comment by l0rd** [comment]
@dkwon17 this page was removed because the problem doesn't exist anymore. That's because with the introduction of the gateway (traefik), single-host is the default and only option. 

Before, with multi-host as the default, workspaces, dashboard and registries had different hostnames (each had its own route/ingress). Thus, accepting to trust the dashboard TLS certificate, when prompted by the browser, was not enough to open a workspace in the browser. Today, with the same hostname for every service, accepting dashboard cert is the only

Another thing to mention is that enterprise policies often forbid developers to install untrusted certificates on their laptop + forbid the use of TLS wildcard certificates for their services: that made Che / CRW completely unusable on these networks.

**Pull request merged**